deploy/mlflow-triton-plugin/mlflow_triton/deployments.py

```diff
@@ -189,8 +190,9 @@ def list_deployments(self):
                 if 's3' in self.server_config:
                     meta_dict = ast.literal_eval(self.server_config['s3'].get_object(
                         Bucket=self.server_config['s3_bucket'],
-                        Key=d['name'] + f'/{_MLFLOW_META_FILENAME}',
-                        )['Body'].read().decode('utf-8'))
+                        Key='/'.join(filter(
```

Comment:
Same here: `filter(None, <list>)` is basically returning `<list>`, so why use if in the first place?